index.js

```diff
@@ -13,9 +13,10 @@ const isObj = o => o !== null && 'object' === typeof o && !Array.isArray(o)
 const isNonEmptyString = str => 'string' === typeof str && str.length > 0
 
 const createClient = (profile, request = _request) => {
-	profile = Object.assign({}, defaultProfile, profile)
-	profile.parseProducts = createParseBitmask(profile)
-	profile.formatProductsFilter = createFormatProductsFilter(profile)
+	profile = Object.assign({
+		parseProducts: createParseBitmask(profile),
+		formatProductsFilter: createFormatProductsFilter(profile)
+	}, defaultProfile, profile)
```

Comment:
Yes, but now the default `parseProduct` won't use the submitted `profile`'s custom products, right? 😄

Reply:
also correct. 😛

Reply:
Ah, I see 😄